airrohr-firmware/platformio.ini

```diff
@@ -44,10 +44,12 @@ lib_deps_generic_external =
   Adafruit SHT31 Library@1.1.6
   ArduinoJson@6.13.0
   DallasTemperature@3.8.0
-  ESP8266_SSD1306@4.1.0
+  thingpulse/ESP8266 and ESP32 OLED driver for SSD1306 displays @ 4.1.0 ; updated SSD1306 url
```

Comment:
@gideonmaina , let's retain the older SSD1306 library.

Reply:
The older SSD1306 does not work as observed in the warning since platform io cannot find a reference to it. Is there a reason for keeping the older SSD1306 library?

Reply:
It does work on my end without an issue @gideonmaina . See the last library on this screenshot.
<img width="734" alt="Screenshot 2023-06-16 at 11 05 39" src="https://github.com/CodeForAfrica/sensors.AFRICA-AQ-sensors-software/assets/28044355/89f6b668-f7dd-4b1d-9e52-a4000befc4d2">

Reply:
Strange because it did not work with a fresh clone. It did not also work on @usmanbiu's Mac. Let's retain it then. Also could either of you or @augustinemwendwa72 test this branch? 
![Screenshot 2023-06-15 at 3 24 13 PM](https://github.com/CodeForAfrica/sensors.AFRICA-AQ-sensors-software/assets/17834362/ccc57e5e-9da7-4a8c-adee-267d9dd4b39f)

Reply:
I've tested it on Mac Intel 2017 @gideonmaina . See the previous screenshot.

Reply:
@Warukira @gideonmaina I have tested GSM_firmware branch, I am waiting for response from @Warukira